### PR TITLE
fix(pivotgrid): add focus state

### PR DESCRIPTION
**Pull request by kikostadinov**
Issue: https://github.com/telerik/kendo-themes/issues/3804

### CODE REVIEW

File: packages/default/scss/pivotgrid/_theme.scss [hunk 1]
```diff
@@ -65,6 +65,9 @@
     .k-pivotgrid-cell.k-state-focus,
     .k-pivotgrid-cell.k-focus,
     .k-pivotgrid-cell:focus,
+    .k-pivotgrid-empty-cell.k-state-focus,
```
Comment (TeyaVes) [inline packages/default/scss/pivotgrid/_theme.scss:68]: let's delete the k-state-focus as we're about to remove all state selectors in the following PR: https://github.com/telerik/kendo-themes/pull/3877

**Review by TeyaVes — CHANGES_REQUESTED** [review]

File: packages/default/scss/pivotgrid/_theme.scss [hunk 2]
```diff
@@ -61,6 +61,9 @@
     // Focus state
     .k-pivotgrid-cell:focus,
     .k-pivotgrid-cell.k-focus,
+    .k-pivotgrid-cell:focus,
```
Comment (Juveniel) [inline packages/default/scss/pivotgrid/_theme.scss:64]: this is duplicated on line 62

**Review by Juveniel — CHANGES_REQUESTED** [review]
Add a test case for the empty cell focus state ( this could probably be achieved in one of the existing tests where states of cells are shown ).

**Review by Juveniel — APPROVED** [review]

**Review by TeyaVes — APPROVED** [review]
